frontend/src/components/NavigationBar.js

```diff
@@ -76,20 +78,38 @@ function NavigationBar({ isAuthenticated }) {
         setLogOut(false);
     }
  
-    const handleChangePasswordOnClick = () => {
+    const handleChangePasswordOnClick = async(event) => {
         setAnchorEl(null);
-        // Triggers change password! 
-    } 
+        event.preventDefault();
+        const refresh_token = cookies["refresh_token"]
+        const username = jwtDecode(refresh_token).username;
+        const res = await axios.put(URL_USER_SVC_UPDATEPASSWORD, { username, newPassword })
```

Comment:
Could i check if we should be using axiosApiInstance to make these requests? Or you already are using it? Thanks

Reply:
I'm not using it here